ansi_color/README.md

```diff
@@ -142,5 +142,5 @@ Here are some links:
 1. <https://docs.rs/anstyle-query/latest/anstyle_query/fn.term_supports_ansi_color.html>
 1. <https://crates.io/crates/anstyle-query>
 1. <https://docs.rs/supports-color/2.0.0/supports_color/>
-1. <https://crates.io/crates/ansi_colours>
+1. <https://crates.io/crates/r3bl_ansi_color>
```

Comment:
Thanks for dropping the ansi_colours crate link.
Since r3bl_ansi_color is the crate for "self", I wonder if there is a better way to indicate this?
Meaning that `https://crates.io/crates/r3bl_ansi_color` is the crate for the code in the `ansi_color` folder itself.